### PR TITLE
simplesamlphp-module-attrauthcomanage_theme_changes

### PR DESCRIPTION
**Pull request by ioigoume**
Enable OrgIdentity customizations for attrauthcomanage-module-simplesamlphp

### CODE REVIEW

File: themes/ssp/attrauthcomanage/noty.tpl.php [hunk 1]
```diff
@@ -77,7 +77,9 @@
     <?php if (!empty($this->data['noty']['icon'])) : ?>
     <img class="gif-box" src="resources/icons/<?= $this->data['noty']['icon'] ?>">
     <?php endif; ?>
-    <p><?= $this->t('{themeegi:attrauthcomanage_noty:title}') ?></p>
+    <?php if(!empty($this->data['noty']['title'])): ?>
```
Comment (nikosev) [inline themes/ssp/attrauthcomanage/noty.tpl.php:80]: Replace line with `<?php if (!empty($this->data['noty']['title'])) : ?>`

File: themes/ssp/attrauthcomanage/noty.tpl.php [hunk 2]
```diff
@@ -77,7 +77,9 @@
...
+    <p><?= $this->t('{attrauthcomanage:noty:' . $this->data['noty']['title'] . '}') ?></p>
```
Comment (nikosev) [inline themes/ssp/attrauthcomanage/noty.tpl.php:81]: Is `title` renamed to `resend_confirmation_email`? If yes, this should be changed as well

File: dictionaries/attrauthcomanage_noty.definition.json [hunk 3]
```diff
@@ -1,27 +1,36 @@
...
+  "org_identity_nologin_description": {
+    "en": "The identifier <b>- %ORGID% -</b> is not a valid authenticator. Please contact <a href=\"mailto:checkin-support@mailman.egi.eu\">our support</a> and make sure to include the error message above."
+  },
+  "org_identity_removed_banner": {
+    "en": "Organization Identity Removed"
+  },
+  "org_identity_removed_description": {
+    "en": "The identifier <b>- %ORGID% -</b> is not a valid authenticator. The subscription from <b>%AUTHNAUTHORITY%</b> has been Removed.<br>Please contact <a href=\"mailto:checkin-support@mailman.egi.eu\">our support</a> and make sure to include the error message above."
   },
   "petition_user_notify": {
```
Comment (nikosev) [inline dictionaries/attrauthcomanage_noty.definition.json:35]: Sort `petition_*` keys as followed

1. `petition_info`
2. `petition_invalidate_msg`
3. `petition_token_expired`
4. `petition_token_expires`
5. `petition_user_notify`

File: themes/ssp/attrauthcomanage/noty.tpl.php (hunk 2 above)
Comment (ioigoume) [reply themes/ssp/attrauthcomanage/noty.tpl.php:81]: `resend_confirmation_email` is the key pulled from the dictionary that will be then found in the `title` key of the noty configuration

**Review by nikosev — CHANGES_REQUESTED** [review]
Please check the comments

File: themes/ssp/attrauthcomanage/noty.tpl.php (hunk 1 above)
Comment (ioigoume) [reply themes/ssp/attrauthcomanage/noty.tpl.php]: Done

File: dictionaries/attrauthcomanage_noty.definition.json (hunk 3 above)
Comment (ioigoume) [reply dictionaries/attrauthcomanage_noty.definition.json]: Done

**Review by nikosev — APPROVED** [review]
LGTM

**Pull request merged**